adapter/persistence/persistence.go

```diff
@@ -0,0 +1,12 @@
+package persistence
+
+import "google.golang.org/appengine/datastore"
+
+// DatastoreHandler
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported type DatastoreHandler should be of the form "DatastoreHandler ..." (with optional leading article)